schema/v1.1.0/common.yaml

```diff
@@ -785,11 +828,63 @@ types:
   VersionString:
     description: A version number (only major, minor versions)
     base: float
+    minimum_value: 0
 
   URLorS3URI:
     description: A URL or S3 URI
     base: str
-    pattern: '^(https?|s3)://.*$'
+    # TODO: Temporary pattern allowing cryoetportal-rawdatasets-dev, but will be removed in the future
+    pattern: '^(((https?|s3)://)|cryoetportal-rawdatasets-dev).*$'
+
+  ORCID:
+    description: A unique, persistent identifier for researchers, provided by ORCID.
+    base: str
+    pattern: '[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$'
+
+  BTO_ID:
+    description: A BRENDA Tissue Ontology identifier
+    base: str
+    pattern: '^BTO:[0-9]{7}$'
+
+  CL_ID:
+    description: A Cell Ontology identifier
+    base: str
+    pattern: '^CL:[0-9]{7}$'
+
+  GO_ID:
+    description: A Gene Ontology identifier
+    base: str
+    pattern: '^GO:[0-9]{7}$'
+
+  ONTOLOGY_ID:
+    description: An ontology identifier
+    base: str
+    pattern: '^[A-Z]+:[0-9]+$'
+
+  DOI:
+    description: A Digital Object Identifier
+    base: str
+    pattern: '^(doi:|https://doi\.org/)?10\.[0-9]{4,9}/[-._;()/:a-zA-Z0-9]+$'
+
+  DOI_LIST:
+    description: A list of Digital Object Identifiers
+    base: str
+    pattern: '^(doi:|https://doi\.org/)?10\.[0-9]{4,9}/[-._;()/:a-zA-Z0-9]+(\s*,\s*(doi:|https://doi\.org/)?10\.[0-9]{4,9}/[-._;()/:a-zA-Z0-9]+)*$'
```

Comment:
We should only allow the doi ids and not allow the links, as the frontend converts these ids to links.

Reply:
Fixed!